src/Request.php

```diff
@@ -611,9 +611,19 @@ public function execute() {
 			$response = apply_filters( 'pre_graphql_execute_request', null, $this );
 
 			if ( null === $response ) {
+
+				/**
+				 * Allow the query string to be determined by a filter. Ex, when params->queryId is present, query can be retrieved.
+				 */
+				$query = apply_filters(
+					'pre_graphql_execute_query_params',
```

Comment:
@markkelnar so the `pre` naming convention indicates filters that run _before_ something and if a response is returned other than null, don't proceed with the normal execution. 

This appears to be just a "normal" filter and not a "pre" filter. 

I think we should consider changing this filter name to `graphql_execute_query_params`